game.po

```diff
@@ -124,92 +124,92 @@ msgstr "NOT IN GAME - Interpolate tangent: average previous track and current tr
 #v1.9.17
 msgctxt "editor_tip_cont_cancel_click"
 msgid "Click to cancel"
-msgstr "Click para cancelar"
+msgstr "Zum Abbrechen klicken"
 
 #v1.9.17
 msgctxt "editor_tip_cont_cancel_dbl_click"
 msgid "Double click to cancel"
-msgstr "Doble click para cancelar"
+msgstr "DOppelklick zum Abbrechen"
```

Comment:
Should be "Doppelklick [...]"

Reply:
Resolved with 0cea2856a245fe98db0b586f1e5d4f299ac7771d